packages/e2e/cypress/e2e/run/actions-pipelinerun.cy.js

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2022 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+const namespace = 'e2e-actions';
+describe('Edit and run Pipeline Run', () => {
+  before(() => {
+    cy.exec('kubectl version --client');
+    cy.exec(`kubectl create namespace ${namespace} || true`);
+  });
+
+  after(() => {
+    cy.exec(`kubectl delete namespace ${namespace} || true`);
+  });
+
+  it('should create pipelinerun on edit and run', function () {
+    // given pipeline
+    const uniqueNumber = Date.now();
+    const pipelineName = `sp-${uniqueNumber}`;
+    const pipeline = `apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: ${pipelineName}
+  namespace: ${namespace}
+spec:
+  tasks:
+    - name: hello
+      taskSpec:
+        steps:
+          - name: echo
+            image: busybox
+            script: |
+              #!/bin/ash
+              echo "Hello World!"
+    `;
+    cy.exec(`echo "${pipeline}" | kubectl apply -f -`);
+    // when create first pipeline run
+    cy.visit(
+      `/#/pipelineruns/create?namespace=${namespace}&pipelineName=${pipelineName}`
+    );
+    cy.get('[id=create-pipelinerun--namespaces-dropdown]').should(
+      'have.value',
+      namespace
+    );
+    cy.get('[id=create-pipelinerun--pipelines-dropdown]').should(
+      'have.value',
+      pipelineName
+    );
+    cy.contains('button', 'Create').click();
+
```

Comment:
We had problems with flakiness in the 'create' e2e so made some changes there, we'll need to do the same here. Specifically the following 2 changes:

- after clicking Create, ensure navigation has completed
  ```js
  cy.contains('h1', 'PipelineRuns');
  ```
- update query to specify an anchor tag as it was occasionally clicking on a matching span instead which caused the test to fail, also no need for `parent()` this way
  ```js
  cy.contains('a', `${pipelineName}-run`).click();
  ```